Formula/formula_renames.json

```diff
@@ -12,5 +12,8 @@
   "orfeo-54": "orfeo5",
   "qgis-214": "qgis2-ltr",
   "qt-mysql": "qt4-mysql",
-  "saga-gis": "saga-gis-lts"
+  "saga-gis": "saga-gis-lts",
+  "qgis2-ltr": "qgis-ltr@2.14",
```

Comment:
why not also to 2.18?

Reply:
yes, this is just a test to see if Circle CI is fixed

Reply:
ok, thanks!

Reply:
I'm in it, I'll make some changes.
It seems that it does not find the bottles in Circle CI, that's why it does not rename them.